GoogleForCreators/web-stories-wp
# Shopping: Provider Dropdown

## Context

For shopping we want to implement a dropdown to choose between existing providers

see: https://github.com/GoogleForCreators/web-stories-wp/issues/11265#issue-1204331478



## Summary

Adds shopping provider dropdown with the following options:
- None
- WooCommerce
- Shopify

![provider](https://user-images.githubusercontent.com/62242/164677092-0493b1c8-94ea-4fe6-a10c-43c9a2f203e9.gif)




## Relevant Technical Choices

Note: I'm also updated the code to show or hide Shopify fields based on the provider selected.



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Ensure the shopping feature is enabled
2. Visit settings
3. Input the shopping provider 
4. Reload the page to ensure the dropdown reloads with the correct value


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11265
